dd-sdk-android/README.md

```diff
@@ -152,6 +152,18 @@ In Java you can do so as follow:
     );
 ```
 
+### Setting the Library's verbosity
+
+If you need to get information about the Library, you can set the verbosity
```

Comment:
> "If you need to get information about the Library (...)" 

it doesn't sound much detailed ("get information"). Maybe something like:

> "If you want to see in Logical how your logs are processed by the Library (...)".

Feel free to disagree 🙂.

Reply:
Well the thing is that this won't apply only to the logs, but eventually to traces, metrics etc…